service/user/service.go

```diff
@@ -0,0 +1,121 @@
+package user
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+	"github.com/stasBigunenko/monorepa/pkg/storage/newStorage"
+)
+
+type UsrService struct {
+	storage newStorage.NewStore
+}
+
+func NewUsrService(s newStorage.NewStore) *UsrService {
+	return &UsrService{
+		storage: s,
+	}
+}
+
+func (u *UsrService) Get(_ context.Context, id uuid.UUID) (model.UserHTTP, error) {
+
+	res, err := u.storage.Get(context.Background(), id)
+	if (res == nil && err == nil) || err != nil {
```

Comment:
`If err != nil` can be other errors. Add one more if.

Reply:
done